### PR TITLE
docs: Added nested workspaces under troubleshooting

### PR DESCRIPTION
**Pull request by NicholasLYang**
### Description

We've had a couple people come to us with issues caused by nested workspaces. I figured we should call this out as unsupported behavior.

### Testing Instructions

Closes TURBO-1569

### CODE REVIEW

File: docs/pages/repo/docs/troubleshooting.mdx [hunk 1]
```diff
@@ -75,6 +75,11 @@ directions even if you aren't planning to make a contribution.
 
 ## Common Pitfalls
 
+### Nested workspaces
+
+Turborepo does not support nested workspaces. Therefore, if you have a workspace that contains another workspace,
```
Comment (NicholasLYang) [inline docs/pages/repo/docs/troubleshooting.mdx:80]: Should we go into more detail as to what a nested workspace is?

Comment (mehulkar) [reply docs/pages/repo/docs/troubleshooting.mdx:80]: Or link to another page in handbook that explains what it is

**Review by mehulkar — APPROVED** [review]

File: docs/pages/repo/docs/troubleshooting.mdx (hunk 1 above)
Comment (NicholasLYang) [reply docs/pages/repo/docs/troubleshooting.mdx:80]: Do we have such a page? It might just be worth fleshing out here

Comment (mehulkar) [reply docs/pages/repo/docs/troubleshooting.mdx:80]: Not sure, just thinking that it's a general monorepo concept and as such can be defined somewhere else. We can do the easy thing and just flesh out here too, just a suggestion

**Pull request merged**